cmd/kepctl/main.go

```diff
@@ -0,0 +1,37 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+	"k8s.io/enhancements/pkg/kepctl"
+)
+
+func main() {
+	cmd, err := buildMainCommand()
+	if err != nil {
+		fmt.Println(err)
```

Comment:
should we print the error to stderr here?

Reply:
+1 thanks.